core/state/statedb.go

```diff
@@ -992,35 +992,37 @@ func (s *StateDB) PrepareAccessList(sender common.Address, dst *common.Address,
 
 // AddAddressToAccessList adds the given address to the access list
 func (s *StateDB) AddAddressToAccessList(addr common.Address) {
-	if s.accessList.AddAddress(addr) {
-		s.journal.append(accessListAddAccountChange{&addr})
+	addrBytes := addr.Bytes20()
+	if s.accessList.AddAddress(addrBytes) {
```

Comment:
No need to declare this separately on this and line 1003. If you push again I'd fix it, but fine as is.

Reply:
The function is different, so the addrBytes on line 995 is out of scope by line 1003